jsapp/js/components/processing/analysis/responseForms/selectOneResponseForm.component.tsx

```diff
@@ -72,14 +72,16 @@ export default function SelectOneResponseForm(props: SelectOneResponseFormProps)
 
   return (
     <ResponseWrapper uuid={props.uuid} onClear={() => setResponse('')}>
```

Comment:
I know this is in another PR, but the clear button placement was a bit confusing to me. There are two question definition related buttons (edit and delete) and between them is a question response related button (clear). Maybe @tesster7 should help us here?

I would expect to have tooltips there at least to make it less confusing :)

The UX is probably better if you're not project manager, because you only have clear.